tests/k8s_testcases/example_policies/services1/services1-scheme.yaml

```diff
@@ -0,0 +1,102 @@
+namespaceList: ../../example_podlist/services1/ns_list.yaml
+podList: ../../example_podlist/services1/
+
+networkConfigList:
+  - name: ip
+    networkPolicyList:
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np0
+    networkPolicyList:
+      - services1-networkpolicy0.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np1
+    networkPolicyList:
+      - services1-networkpolicy1.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np2
+    networkPolicyList:
+      - services1-networkpolicy2.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+queries:
+  - name: connectivity_map
+    connectivityMap:
+    - ip
+    - np0
+    - np1
+    - np2
+    expected: 0
+    outputConfiguration:
+      outputFormat: txt
+      fwRulesRunInTestMode: false
+    expectedOutput: ../expected_output/services1_connectivity_map.txt
+
+  - name: sanity
+    sanity:
+      - np1
+    expected: 2
+
+  - name: compare_np1_and_np2
+    twoWayContainment:
+      - np1
+      - np2
+    expected: 1
+
+  - name: np1_and_np2_are_not_strongly_equivalent
+    strongEquivalence:
+      - np1
+      - np2
+    expected: 0
+  - name: np1_and_np1_are_strongly_equivalent
+    strongEquivalence:
+      - np1
+      - np1
+    expected: 0
+
+  - name: compare_specific_policies_in_np1_and_np2
+    equivalence:
+      - np1/ingress-nginx/allow-all-within-namespace
+      - np2/ingress-nginx/allow-all-within-namespace
+      - np1/ingress-nginx/allow-webhook-via-ingress
+    expected: 2
+
+  - name: redundancy_np1_and_np2
+    redundancy:
+      - np1
+      - np2
+    expected: 4
+
+  - name: pairwiseInterferes_np1_and_np2
+    pairwiseInterferes:
+      - np1
+      - np2
+    expected: 1
+
+  - name: not_empty
+    emptiness:
```

Comment:
For vacuity and emptiness  queries - how should we handle configs with Ingress only? 
Should we ignore Ingress resources for these queries?

Reply:
For emptiness, I agree that Ingress only config cannot be empty - skipping these queries with warning.
For vacuity, I think theoretically there may be a case when Ingress backend covers all possible pods, and thus does not limit any connection.

Reply:
It still limits connections from ingress controller pod to only HTTP connections (not all connections for any protocol are allowed from this pod).